src/server/test.rs

```diff
@@ -267,7 +267,7 @@ fn delete_collection() {
         http::Method::DELETE,
         "/1.5/42/storage/bookmarks",
         None,
-        Some("0.00"),
+        None, // deleting a collection may return a tombstone rather than "0.00"
```

Comment:
This check might as well switch to `test_endpoint_with_response` w/ the exact same `result > start` closure in the other 2 checks in this test

Reply:
Not quite, but I did chase down an inconsistency with how mysql and spanner were dealing with this.